test/express.test.js

```diff
@@ -0,0 +1,65 @@
+'use strict'
+
+const test = require('tap').test
+const express = require('express')
+
+const inject = require('../index')
+
+test('works with express with callback', (t) => {
+  t.plan(8)
+
+  const app = express()
```

Comment:
can we potentially automatically detect this?

Reply:
Well, `app` contains a bunch of express-specific properties, so we can probably check for some of them, but not sure how reliable it would be. 

`app.handle`
`app.lazyrouter`
`app.request`
`app.response`

are some of them, do you think they're good enough for a guess?

Reply:
probably not. are app.request and response the prototypes of the req and res? We might be able to auto detect with these.

Reply:
Yep
```js
app.request instanceof http.IncomingMessage === true
app.response instanceof http.ServerResponse === true
```

Reply:
Do you think these two are enough to decide if it's express?
I was leaning towards `inject.express(app, '/path')`

Reply:
Let's do `inject.express(app, '/path')`

Reply:
No need, code is ok